sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -3648,4 +3648,7 @@ INSERT INTO schema_version (version_number, comment) VALUES (166, 'Add pathogenT
 -- 2019-11-04 Fill samples with accurate pathogenTestResultChangeDate #1349
 UPDATE samples SET pathogentestresultchangedate = (SELECT testdatetime FROM pathogentest WHERE pathogentest.sample_id = samples.id ORDER BY pathogentest.testdatetime DESC LIMIT 1);
 
-INSERT INTO schema_version (version_number, comment) VALUES (167, 'Fill samples with accurate pathogenTestResultChangeDate #1349');
\ No newline at end of file
+INSERT INTO schema_version (version_number, comment) VALUES (167, 'Fill samples with accurate pathogenTestResultChangeDate #1349');
+
+-- 2019-11-06 Bed-side lab testing #1109
+ALTER TABLE samples ADD COLUMN samplepurpose  varchar(255) DEFAULT 'EXTERNAL' NOT NULL;
```

Comment:
@opatajoshua Remember to add a schema_version entry after the changes. I will take care of this now.